### ISSUE TITLE
is this project dead?

### ISSUE THREAD

**Issue by terminaleffect** [shycats/VirtuCameraBlender#28]
title

**Comment by festivities** [comment]
I paid for VirtuCamera before it went free so it'd be a huge disappointment if this were the case.

I honestly don't think this add-on would've completely broke if the main module wasn't compiled with Cython. I attempted modifying the add-on myself until 5 seconds into the tinkering, I realized that the .pyd file being linked against Python 3.10 and Blender 4.1 migrating to Python 3.11 is the problem itself. A bandaid fix by outsiders is virtually impossible, if not more trouble than it's worth.

If this product's indeed abandoned, I really hope the devs put the rest of the source code out.

**Comment by hoodwinkedfool** [comment]
@shycats I would rebuy the app at this point I'm so desperate

**Comment by hsa013** [comment]
@shycats 
Could you please advise or comment whether this project will be updated to newer versions of Blender?
As a paying customer I would love to use this fantastic app in the future.
Using previous Blender versions is not viable as it is impossible to open 4.x files in 3.x in full compatibility.  Obviously it is not viable to work in outdated 3.x either.
Many thanks for your feedback.

**Comment by Mitchellja13** [comment]
Seriously, if this was a paid app I would pay for the new version :(

**Comment by aphaits** [comment]
Can this app record video while also tracking camera? I'm looking to composite 3d on video and having the phone also grab video recording might be ideal if it can simultaneously track the camera virtually.

**Comment by BatsLog** [comment]
Clearly, not coming back. Any alternatives?

**Comment by eisenbruch** [comment]
Im out of the loop and kinda even forget exactly what this project does/did, but does [Tracky](https://github.com/Shopify/tracky) solve your issue?

**Comment by hoodwinkedfool** [comment]
> Clearly, not coming back. Any alternatives?

Unfortunately no. All alternatives I’ve found are just tracking apps. Nothing that does everything in Blender. My only suggestion is to use 3.6 for the time being. I recently had to and it was a huge pain but it got the job done. I had to make primitives of everything in my scene as proxies for the stuff that didn’t work in the older version but at least I could use the app.
I wonder what happened to the developer.

**Comment by GeoPanago** [comment]
Since it's been a very long time without a solution I found my own. I made a bridge between Blender 3.6 and 4.4. Two addons, one in each that allow me to replicate the movement of a selected camera in 3.6 to a target camera in 4.4. For me it works without any lag in real time with VirtuCamera enabled in 3.6. The viewport on my phone using this workaround cannot display Blender 4.4 of course but I can definitely live with it.

**Comment by hoodwinkedfool** [comment]
> Since it's been a very long time without a solution I found my own. I made a bridge between Blender 3.6 and 4.4. Two addons, one in each that allow me to replicate the movement of a selected camera in 3.6 to a target camera in 4.4. For me it works without any lag in real time with VirtuCamera enabled in 3.6. The viewport on my phone using this workaround cannot display Blender 4.4 of course but I can definitely live with it.

So you open two instances of Blender? The signal still sends to the target camera in 4.4? If so that is amazing thank you!

**Comment by GeoPanago** [comment]
> > Since it's been a very long time without a solution I found my own. I made a bridge between Blender 3.6 and 4.4. Two addons, one in each that allow me to replicate the movement of a selected camera in 3.6 to a target camera in 4.4. For me it works without any lag in real time with VirtuCamera enabled in 3.6. The viewport on my phone using this workaround cannot display Blender 4.4 of course but I can definitely live with it.
> 
> So you open two instances of Blender? The signal still sends to the target camera in 4.4? If so that is amazing thank you!

Yes, exactly. Both are open, 3.6 has only one camera in the scene which sends to 4.4. In 4.4 I choose which camera will be controlled.

**Comment by BatsLog** [comment]
> > > Since it's been a very long time without a solution I found my own. I made a bridge between Blender 3.6 and 4.4. Two addons, one in each that allow me to replicate the movement of a selected camera in 3.6 to a target camera in 4.4. For me it works without any lag in real time with VirtuCamera enabled in 3.6. The viewport on my phone using this workaround cannot display Blender 4.4 of course but I can definitely live with it.
> > 
> > 
> > So you open two instances of Blender? The signal still sends to the target camera in 4.4? If so that is amazing thank you!
> 
> Yes, exactly. Both are open, 3.6 has only one camera in the scene which sends to 4.4. In 4.4 I choose which camera will be controlled. I measured the added latency it is very low, around 0.15 ms on average with some spikes around 1ms

You are amazing! Thank you.
